### PR TITLE
Damola o branch

### LINKED ISSUE #153: Parallel processing
* **Is your feature request related to a problem? Please describe.**
*try out joblib

* **Describe the solution you'd like**

* **Describe alternatives you've considered**

* **Additional context**

### PR DESCRIPTION
**Pull request by AAD-Damola-Ogunwoolu**
## Proposed changes
implementing parallel processing
Describe the big picture of your changes here to make it clear what your pull request will change. Make sure to explain the **what** e.g. the functionality it changes not the **how** e.g. what code you have written.  If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [x ] New feature (change which adds functionality)
- [ ] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x ] I have checked that my changes have not broken any other functionality
- [x ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [ ] Your code is clear and easy to understand
- [ ] You have documented any new features
- [ ] They can run the code you have written
- [ ] Your changes do not break any other functionality

## Who is reviewing this PR?
Bashir 
Tag your reviewer here to ensure they get a notification!
@AAD-Bashir-Ali

### CODE REVIEW

File: manon-seguin [hunk 1]
```diff
@@ -0,0 +1 @@
+testing
```
Comment (DIPAD-Fran-Bryden) [inline manon-seguin:1]: Great bit of code

File: manu text file [hunk 2]
```diff
@@ -0,0 +1,5 @@
+mock text file
+
+adding in a few lines below
+line 4 added here
+line 5 here
```
Comment (DIPAD-Fran-Bryden) [inline manu text file:5]: I don't think line 5 is necessary

File: read_ons_bulletin.R [hunk 3]
```diff
@@ -22,7 +22,7 @@ read_release <- function(){
 read_dates <- function(){
 
   #Read all HTML
-  pg <- read_html("https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/healthandwellbeing/bulletins/coronavirusandthesocialimpactsongreatbritain/latest")
+  pg <- read_html("https://www.ons.gov.uk/peoplepopulationandcommunity/")
```
Comment (DIPAD-Fran-Bryden) [inline read_ons_bulletin.R:25]: I think this is broken

**Review by DIPAD-Fran-Bryden — APPROVED** [review]
Generally well written

File: damola-o.txt
Comment (AAD-Bashir-Ali) [inline damola-o.txt:1]: Great file!

File: manon-seguin
Comment (AAD-Bashir-Ali) [inline manon-seguin:1]: Brilliant

File: report_functions.R [hunk 4]
```diff
@@ -188,7 +191,7 @@ date_cycling <- function(format = "full_month"){
   if(format == "full_month"){
     
     paste0(date_formatter(min(data$date, na.rm = TRUE), abbr_day = FALSE, abbr_month = FALSE), " and ", date_formatter(max(data$date, na.rm = TRUE), abbr_day = FALSE, abbr_month = FALSE))
-  }else{
+  } else{
     paste0(date_formatter(min(data$date, na.rm = TRUE)), " - ", date_formatter(max(data$date, na.rm = TRUE)))
```
Comment (AAD-Bashir-Ali) [inline report_functions.R:196]: Extra space?

**Review by DFT-Bashir-Ali — APPROVED** [review]
Made some slight suggestions but looks great!

**Review by DFT-Bashir-Ali — APPROVED** [review]

**Pull request closed without merging**